src/formatDate.js

```diff
@@ -1,14 +1,62 @@
 'use strict';
 
-/**
- * @param {string} date
- * @param {string[]} fromFormat
- * @param {string[]} toFormat
- *
- * @returns {string}
- */
 function formatDate(date, fromFormat, toFormat) {
-  // write code here
+  const dateToArrow = date.split(fromFormat[fromFormat.length - 1]);
```

Comment:
The variable name 'dateToArrow' is a bit misleading as it may imply some connection with arrows, which is not the case. Consider renaming it to 'dateParts' or 'dateComponents' to better reflect its purpose.